helpers/helpers.go

```diff
@@ -400,18 +400,24 @@ func CheckTokenExpiration(accessToken string) error {
 	switch untilExp := time.Until(expiration); {
 	case untilExp < 0:
 		return fmt.Errorf("the provided access token has expired, please renew it")
-	case untilExp > 0 && untilExp < 24*time.Hour:
-		fmt.Fprintln(
+	case untilExp >= 0 && untilExp < 2*time.Hour:
+		fmt.Fprintf(
 			os.Stderr,
-			"The provided access token expires in",
-			time.Until(expiration).Truncate(time.Second),
+			"WARNING! The provided access token expires in only %d hours and %d minutes.\n",
+			int(untilExp.Hours()), int(untilExp.Minutes())-(int(untilExp.Hours())*60),
+		)
+		fmt.Fprintln(os.Stderr, "Consider renewing the token.")
+	case untilExp >= 2*time.Hour && untilExp < 24*time.Hour:
+		fmt.Fprintf(
+			os.Stderr,
+			"The provided access token expires in %d hours and %d minutes (at %d:%d).\n",
+			int(untilExp.Hours()), int(untilExp.Minutes())-(int(untilExp.Hours())*60), expiration.Hour(), expiration.Minute(),
 		)
 		fmt.Fprintln(os.Stderr, "Consider renewing the token.")
 	default:
-		fmt.Fprintln(
+		fmt.Fprintf(
 			os.Stderr,
-			"The provided access token expires in",
-			time.Until(expiration).Truncate(time.Second),
+			"The provided access token expires on %s.\n", expiration.Format("2006-01-02"),
```

Comment:
This is the same as using 
```
expiration.Format(time.DateOnly)
```

In this case it might be better to be a bit more verbose so that the user knows if the token will expire outside of working hours.
```suggestion
			"The provided access token expires on %s.\n", expiration.Format(time.RFC1123),
```

This will then print the following message:
```
The provided access token expires on Mon, 27 Jan 2025 08:00:22 CET
```

Reply:
agreed,looks neater this way :smile: